scripts/import_file.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
This conflict with the fact that you check for python 2 in `import_from_file`.

Reply:
This file is never run directly, but imported by other modules as a lib.
That is the reason why it support both py2 and py3 for compatiblity.
But it is still good to have this line removed.

Reply:
This py2 check was added for the CI python2 pytest.
Will remove the shebang